Godeps/Godeps.json

```diff
@@ -1888,18 +1888,15 @@
 		},
 		{
 			"ImportPath": "github.com/heketi/heketi/client/api/go-client",
-			"Comment": "v2.1.0-dev-2-gcb07059",
-			"Rev": "cb07059aed8760ef857a58d37598b9e51c36a9ae"
+			"Rev": "28b5cc4cc6d2b9bdfa91ed1b93efaab4931aa697"
 		},
 		{
 			"ImportPath": "github.com/heketi/heketi/pkg/glusterfs/api",
-			"Comment": "v2.1.0-dev-2-gcb07059",
-			"Rev": "cb07059aed8760ef857a58d37598b9e51c36a9ae"
+			"Rev": "28b5cc4cc6d2b9bdfa91ed1b93efaab4931aa697"
 		},
 		{
 			"ImportPath": "github.com/heketi/heketi/pkg/utils",
-			"Comment": "v2.1.0-dev-2-gcb07059",
-			"Rev": "cb07059aed8760ef857a58d37598b9e51c36a9ae"
+			"Rev": "28b5cc4cc6d2b9bdfa91ed1b93efaab4931aa697"
```

Comment:
A bump commit that doesn't actually bump anything but the Godep file doesn't make sense